website/src/pages/PublicRoute.tsx

```diff
@@ -43,7 +43,10 @@ const PublicRoute = () => {
         <Route path="/search" element={<SearchPage />} />
         <Route path="/compare" element={<ComparePage />} />
         <Route path="/macro" element={<MacroPage />} />
-        <Route path="/macrobench/queries/compare" element={<MacroQueriesComparePage />} />
+        <Route
+          path="/macrobench/queries/compare"
+          element={<MacroQueriesComparePage />}
+        />
```

Comment:
Any reason for this change?

Reply:
Seems like this change is done by prettier extension of vscode. This is not required.

Reply:
I have reverted back that change 👍🏻